src/types/index.ts

```diff
@@ -61,4 +61,19 @@ export enum DevicePermissionStatus {
   IN_PROGRESS = 'IN_PROGRESS',
   GRANTED = 'GRANTED',
   DENIED = 'DENIED',
-};
\ No newline at end of file
+};
+
+export enum DeviceLabels {
+  None,
+  Audio,
+  Video,
+  AudioAndVideo,
+};
+
+export enum Device {
```

Comment:
nit: Should this be deviceType?

Reply:
`DeivceType` is a better naming for sure. The question is, here it's an `enum`, and I found in this repo, anything with a 'Type' suffix, is a `type` type. Besides, we already have a `DeviceType`, see below.
```ts
export type DeviceType = {
  deviceId: string;
  label: string;
};
```
Considering these, do we still need to rename it?

Reply:
How about DeviceKind? That aligns with webRTC MediaStream.

Reply:
Sounds great!

Reply:
Just to clarify: a `Type` suffix does not require that it be a TypeScript `type` alias. `Kind` is often a good word, but wanted to avoid an accidental rule forming.